.mvn/maven.config

```diff
@@ -0,0 +1,4 @@
+-Daether.remoteRepositoryFilter.prefixes=true
+-Daether.remoteRepositoryFilter.prefixes.basedir=${session.rootDirectory}/.mvn/rrf/
+-Daether.remoteRepositoryFilter.groupId=true
+-Daether.remoteRepositoryFilter.groupId.basedir=${session.rootDirectory}/.mvn/rrf/
```

Comment:
Why `${session.rootDirectory}` and not `${project.rootDirectory}`?

Reply:
This is a new property introduced in Maven4 (and backported to Maven 3.9) and denotes "root of the project checkout". Basically, latter _depends_ where you invoke Maven (ie `mvn -f plugins/` then `plugins/.mvn/rrf` would be it). The former value is recursively checked and determined, is usable ONLY if can be exactly determined (with presence of `.mvn` directory).

Reply:
See https://issues.apache.org/jira/browse/MNG-7774

Reply:
This is great! We can use it to replace the `air.main.basedir` that we set manually in all of our modules.

Reply:
https://github.com/trinodb/trino/pull/20486